res/controllers/common-hid-packet-parser-test.js

```diff
@@ -0,0 +1,29 @@
+// Manual packing test functions
```

Comment:
It was a bunch of code that was commented out in `common-hid-packet-parser.js` and seemed to be for testing. So I moved it out instead of deleting in case it was useful.

Reply:
As far as I can tell, this doesn't really test anything, it's just a tool for debugging. I'd prefer to move it back to common-hid-packet-parser.js than have this stray file.

Reply:
I don't like to have this stray piece of code in a file I regularly use as a reference.

Reply:
Maybe we should just delete it?

Reply:
Yes. It will still be in the history if somebody needs it ever again